### ISSUE TITLE
TranslatorVersion is missing from Library ELM JSON

### ISSUE THREAD

**Issue by RohitKandimalla** [cqframework/clinical_quality_language#1186]
After generating ELM JSON and ELM XML using CQL-ELM-Translator, Looks like translatorVersion is missing from ELM JSON, but is present in ELM XML.

Example translatorVerison="2.11.0" is available in elm-xml
`<annotation translatorVersion="2.11.0" translatorOptions="EnableAnnotations,EnableLocators,EnableResultTypes,EnableDetailedErrors,DisableListDemotion,DisableListPromotion" xsi:type="a:CqlToElmInfo"/>`

It is expected that ELM JSON should also contain information related to translated version.

Please advice

MADiE team received an HelpDesk ticket regarding the same [BONNIEMAT-1563](https://oncprojectracking.healthit.gov/support/browse/BONNIEMAT-1563)

**Comment by JPercival** [comment]
Hi @RohitKandimalla!

Have you validated that the the ELM was generated with the `EnabledAnnotations` flag enabled?

**Comment by RohitKandimalla** [comment]
I will verify that and get back to you, Is the EnabledAnnotations specific to ELMJson ?

**Comment by RohitKandimalla** [comment]
@JPercival Yes, I verified and ELM is generated using `EnabledAnnotations=true`

**Comment by brynrhodes** [comment]
@RohitKandimalla can you please post a snippet of the JSON that does not have the translator version? I'm wondering if this is related to the other JSON ELM serialization issue?

**Comment by RohitKandimalla** [comment]
Hi Bryn, 

Sorry for the late reply, I was not receiving the notifications.

but Idk how to post a snippet if it is not there, 
so here is the comparison b/w XML and JSON of the ELM, Looks like "Annotations" object itself is missing from JSON. 

I am also attaching the json 
[ELM_With_no_version_invormation.txt](https://github.com/cqframework/clinical_quality_language/files/12832272/ELM_With_no_version_invormation.txt)

![image](https://github.com/cqframework/clinical_quality_language/assets/57495404/36595872-0f64-4eb3-8d67-4a5fb8754014)

**Comment by RohitKandimalla** [comment]
This is a snippet provided by user, I am trying to use translator version 2.9.0, but I can't seem to reproduce it.
{
   "library" : {
      "annotation" : [ {
         "translatorVersion" : "2.9.0", ...

**Comment by ddieppois** [comment]
I tested this translation locally and was able to get the "translatorVersion" : "3.3.0-SNAPSHOT", added to the JSON. I tried it from the ChlamydiaScreening_Common.cql with the added option "-format JSON" added. The value was provided in the JSON.

Could you guys try it again with the latest version of the translator ?

**Comment by mdnazmulkarim** [comment]
Tested it on master and it is now rendering the translatorVersion to Json and Xml. Thus it is closed now.
